tests/performance/scale/tests/create-cluster.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+export INFRA_NAME=$1
+
+export ARTIFACTS_DIR="/tmp/artifacts-${INFRA_NAME}"
+
+infractl create openshift-4-perf-scale "${INFRA_NAME}" --arg master-node-type=n2-standard-16 --description "Perf testing cluster" --download-dir="${ARTIFACT_DIR}"
```

Comment:
Let's change to `n4-standard-16` - it's a newer CPU generation. If they are available.